google-ads-annotation-processing/build.gradle

```diff
@@ -29,3 +29,12 @@ dependencies {
 
 description = 'Google Ads API client library for Java annotation processor'
 
+publishing {
+    publications {
+        maven(MavenPublication) { publication ->
+            configurePom(publication,
+                    "Google Ads API client library for java - main library",
+                    "Main library for the Google Ads API client library for Java")
```

Comment:
Should this description differ from the one for `google-ads`, since this one's for the annotation processing module?

Reply:
DOne